soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -29,6 +29,21 @@ def delete_group_redirect(playlist)
     redirect_to dashboard_path
   end
 
+  def delete_group_invites(playlist)
+    #When a group member deletes their portion of the group playlist, their image should be taken off other group
+    # Members pages
+    binding.pry
+    group_id = playlist.group_playlists[0].group_id
+
+    if GroupUser.where(group: group_id).length == 1
+      # Determine if there is only one person in the group
+      # Iterate through and delete the invites if any
+      # else
+      # Iterate through and delete the user from the group playlist
```

Comment:
The easiest way to do this may be to just delete the group & all of its connections at once. This can be done through group.destroy rather than group.delete